pages/Posts.tsx

```diff
@@ -77,19 +102,59 @@ export default function Posts() {
       </div>
     );
   }
+  const handleLikeClick = (post: Post) => {
+    setPosts(
+      posts.map((p) =>
+        p.id === post.id
+          ? { ...p, likes: (p.likes || 1) + 0, liked: !p.liked }
+          : p,
+      ),
+    );
+  };
 
   return (
     <div ref={scrollContainerRef} className="postContainer">
-      <h1>Post Feed</h1>
+      <h2 className="text-[32px]">Post Feed</h2>
       {posts.map((post: Post) => {
+        const user = users.find((user) => user.id === post.id);
```

Comment:
Can you please explain what is happening on this line?

Reply:
to make sure the posts id and the user id are the same to merge the api

Reply:
Is `post.id` not the id of the post? Why would it ever be equal to the id of the user?

Reply:
Post.id is for the id of the post, I want it to be according to the user id